httpbakery/client.go

```diff
@@ -147,9 +149,9 @@ func relativeURL(base, new string) (*url.URL, error) {
 }
 
 func (ctxt *clientContext) do(req *http.Request, getBody BodyGetter) (*http.Response, error) {
-	log.Printf("client do %s %s {", req.Method, req.URL)
+	logger.Debugf("client do %s %s {", req.Method, req.URL)
 	resp, err := ctxt.do1(req, getBody)
-	log.Printf("} -> error %#v", err)
+	logger.Infof("} -> error %#v", err)
```

Comment:
Might this look a little odd if logging at info level or above, the curly brackets won't match.